test/functional/tests/cache_ops/test_seq_cutoff.py

```diff
@@ -246,9 +247,10 @@ def verify_writes_count(core, writes_before, threshold, io_size, ver_type=Verify
 def prepare(cores_count=1, cache_line_size: CacheLineSize = None):
     cache_device = TestRun.disks['cache']
     core_device = TestRun.disks['core']
-    cache_device.create_partitions([(SEQ_CUTOFF_THRESHOLD_MAX * cores_count
-                                    + Size(5, Unit.GibiByte)).align(0x1000)])
-    partitions = [(SEQ_CUTOFF_THRESHOLD_MAX + Size(10, Unit.GibiByte)).align(0x1000)] * cores_count
+    cache_device.create_partitions(
+        [(SEQ_CUTOFF_THRESHOLD_MAX * cores_count + Size(5, Unit.GibiByte)).align(0x1000)])
```

Comment:
You left `.align(0x1000)` which You renamed to `align_down`.

Reply:
Good catch!